script.js

```diff
@@ -1,4 +1,68 @@
+// initialise first mode asking input
+var mode = 'need input';
+
+// two dice array for ramdom dice roll
+var diceOne = [];
+var diceTwo = [];
+
+// player 1 and 2 number
+var player1Dice = [];
+var player2Dice = [];
+
+// random dice generator for two dice
+var generateDiceNum = function () {
+  return Math.ceil(Math.random() * 6);
+};
+
+// function to concat first dice number
+function concatDiceNum1() {
+  return (diceOne[diceOne.length - 1]).toString() + (diceTwo[diceTwo.length - 1]).toString();
+}
+// function to concat second dice number
+function concatDiceNum2() {
+  return (diceTwo[diceTwo.length - 1]).toString() + (diceOne[diceOne.length - 1]).toString();
+}
```

Comment:
Can these functions be combined into 1? E.g. `concatDiceNums(firstDiceNum)`, where `firstDiceNum` specifies whether the 1st or 2nd dice should go first?